projects/plugins/jetpack/jetpack.php

```diff
@@ -172,3 +172,14 @@ function jetpack_admin_missing_autoloader() {
 
 // Require everything else, that is not loaded via the autoloader.
 require_once JETPACK__PLUGIN_DIR . 'load-jetpack.php';
+
+/*
+ * Enable upgrade nudge for Atomic sites.
+ * This feature is false as default,
+ * so let's enable it through this filter.
+ *
+ * More doc: https://github.com/Automattic/jetpack/tree/master/projects/plugins/jetpack/extensions#upgrades-for-blocks
+ */
+if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) {
+	add_filter( 'jetpack_block_editor_enable_upgrade_nudge', '__return_true' );
+}
```

Comment:
Would it be possible to keep this closer to the code, maybe by adding a check before the filter is defined in `projects/plugins/jetpack/class.jetpack-gutenberg.php`?

Reply:
👍 lets try it

Reply:
Thanks, Jeremy. Tackled here 991d953529f6f9ac519c268c8a6b492331c448f0.